lib/src/over_react_test/string_util.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// Utility function to convert a `SCREAMING_SNAKE` [str] into a `spinal-case` equivalent.
+String screamingSnakeToSpinal(String str) {
```

Comment:
Is this one-liner really worth including?

```dart
String screamingSnakeToSpinal(String str) => str.replaceAll('_', '-').toLowerCase();
```

I'm not opposed to leaving it in, but it seems extraneous.